app/src/main/res/layout/fragment_consents_list.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
move `<!--` to the new line

Reply:
then should then go through all the files and bring them to a general view.